internal/controller/managedzone_controller.go

```diff
@@ -156,7 +157,7 @@ func (r *ManagedZoneReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	if err != nil {
 		return ctrl.Result{}, err
 	}
-	log.Log.Info("Reconciled ManagedZone", "managedZone", managedZone.Name)
+	logger.Info("Reconciled ManagedZone")
```

Comment:
why don't we need the managedzone name here, but we do need it in some of the others?

Reply:
like here: https://github.com/Kuadrant/dns-operator/pull/152/files#diff-5882be9025c1bbbef34525f6ffba174154eaac4c288cd24fb8ebb968a287bf84R220

Reply:
We don't will update those to remove the duplicate key/values

Reply:
Should all be updated now to remove adding the details of the current resource at individual log statements. That is already being done as part of the logger setup in `Reconcile`. Further improvements to the logging and what metadata is included and where will be done as part of  https://github.com/Kuadrant/dns-operator/issues/118